Kernel/Random.h

```diff
@@ -67,9 +69,10 @@ class FortunaPRNG {
 
         // generate a new key
 
-        for (size_t i = 0; i < KeySize; i += block_size) {
+        for (size_t i = 0; i < KeySize/8; i += block_size) {
```

Comment:
I'm getting a feeling that you're not running clang-format?
Please run clang-format on these (and if possible, squash the changes to their respective commits).

Reply:
All files are clang formatted (at lease at HEAD) Except for this file in this commit. i'm not sure that I can format and squash these changes, because they no longer exist.

Reply:
That's fine then.
I was just making sure :)